### ISSUE TITLE
Broken documentation for templates

### ISSUE THREAD

**Issue by mmarin11** [peering-manager/peering-manager#277]
Dear peering-manager community

Are there some templates examples for Arista or Juniper we can use as examples? Documentation has some links but links sees to be broken

**Comment by respawner** [comment]
Documentation do exist but example pages are broken. It looks like files are not found during compilation:

```
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /home/docs/checkouts/readthedocs.org/user_builds/peering-manager/checkouts/latest/_build/html 
WARNING -  A relative path to 'templates/index.md' is included in the 'nav' configuration, which is not found in the documentation files 
WARNING -  A relative path to 'templates/arista-eos.md' is included in the 'nav' configuration, which is not found in the documentation files 
WARNING -  A relative path to 'templates/cisco-iosxr.md' is included in the 'nav' configuration, which is not found in the documentation files 
WARNING -  A relative path to 'templates/juniper-junos.md' is included in the 'nav' configuration, which is not found in the documentation files 
WARNING -  A relative path to 'templates/peering-request-email.md' is included in the 'nav' configuration, which is not found in the documentation files
```

**Comment by mngan** [comment]
This looks to be caused by confusion of having a directory called `templates` at the top level as well as in the `docs` directory. If you rename the one in the `docs` directory (e.g `docs/templates` to `docs/tmpl`), the build succeeds. I tested this out by running `pip install mkdocs` and playing around with `mkdocs serve` or `mkdocs build`. It also looks like the `docs/templates/index.md` also needs the references to the model documentation to be updated.

**Comment by respawner** [comment]
I suspected that the issue was from the templates directory. I tried to do the same thing but the build was still failing on my local instance. I probably missed something. I'm not sure what's the best plan from here: rename the directory in `docs/` or opening an issue about this as it looks like a regression. This used to work and it broke someday…

**Comment by mngan** [comment]
Looks like this [line](https://github.com/mkdocs/mkdocs/blob/1ad6a91e3ff04a7f61b3bf376dbca84c5169279a/mkdocs/structure/files.py#L218) is the problem. The `templates` directory is special, and is explicitly excluded.

**Comment by mngan** [comment]
This should be fixed in [PR#285](https://github.com/peering-manager/peering-manager/pull/285)
